### PR TITLE
[addons] use correct revision for kodi-platform

### PR DESCRIPTION
**Pull request by olafhering**
This is the master branch, so use also HEAD of development for dependencies.

Signed-off-by: Olaf Hering <olaf@aepfle.de>

## Types of change

- [x] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [ ] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [ ] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **None of the above** (please explain below)

## Checklist:

- [ ] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [ ] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

File: cmake/addons/depends/common/kodi-platform/kodi-platform.txt [hunk 1]
```diff
@@ -1 +1 @@
-kodi-platform https://github.com/xbmc/kodi-platform e8574b883ffa2131f2eeb96ff3724d60b21130f7
+kodi-platform https://github.com/xbmc/kodi-platform master
```
Comment (Rechi) [inline cmake/addons/depends/common/kodi-platform/kodi-platform.txt]: no, use the exact commit revision

Comment (olafhering) [reply cmake/addons/depends/common/kodi-platform/kodi-platform.txt]: What, no release branch? Come on ...

This PR will stay as it is.

Comment (wsnipex) [reply cmake/addons/depends/common/kodi-platform/kodi-platform.txt]: then it won't get merged.

**Comment by olafhering** [comment]
Just to get it off the table.

**Pull request merged**